code/__defines/items_clothing.dm

```diff
@@ -64,7 +64,7 @@
 #define BLOCKHAIR       0x40    // Hides the user's hair, facial and otherwise.
 
 //This flag applies to gloves, uniforms, shoes, masks, ear items, glasses
-#define ALWAYSDRAW	32//If set, this item is always rendered even if its slot is hidden by other clothing
+#define ALWAYSDRAW	0x80//If set, this item is always rendered even if its slot is hidden by other clothing
```

Comment:
what do you mean?
It DID conflict, thats why i had to change it. Worn holder mobs were hiding hair